sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -28,6 +28,9 @@ set(UR_BUILD_TESTS "${SYCL_UR_BUILD_TESTS}" CACHE BOOL "" FORCE)
 # UR tests require the examples to be built
 set(UR_BUILD_EXAMPLES "${SYCL_UR_BUILD_TESTS}" CACHE BOOL "" FORCE)
 
+option(SYCL_UR_FORMAT_CPP_STYLE "Format code style of UR C++ sources" OFF)
```

Comment:
Why do we need this? Can't we just pass UR_FORMAT_CPP_STYLE  when configuring SYCL?